src/jquery.addressfield.js

```diff
@@ -70,7 +71,10 @@
         url: configs.json,
         async: configs.async,
         success: function (data) {
-          $.fn.addressfield.binder.call($container, configs.fields, $.fn.addressfield.transform(data));
+          transformedData = $.fn.addressfield.transform(data);
+
+          $.fn.addressfield.initCountries.call($container, configs.fields.country, transformedData);
```

Comment:
- [ ] This looks great, @angrytoast.  Only nit is that I'd like to see that country initialization is also performed via the AJAX mechanism callback.  Believe all of your existing tests check the inline JSON.